--- a/fluid/Fl_Type.h	2014-07-24 17:15:14.000000000 -0700
+++ b/fluid/Fl_Type.h	2014-07-24 17:15:37.000000000 -0700
@@ -36,7 +36,7 @@
 class Fl_Type {
 
   friend class Widget_Browser;
-  friend Fl_Widget *make_type_browser(int,int,int,int,const char *l=0);
+  friend Fl_Widget *make_type_browser(int,int,int,int,const char *l);
   friend class Fl_Window_Type;
   virtual void setlabel(const char *); // virtual part of label(char*)
 
